### ISSUE TITLE
Full scrape fails while limited one succeeds

### ISSUE THREAD

**Issue by Popolechien** [openzim/zimit#179]
A user ran a successful (?) scrape of ivelt.com [via youzim.it](https://farm.youzim.it/pipeline/69536ab06dc2ee44b109d346) and asked if getting a fuller copy would be doable. I obliged and created https://farm.openzim.org/recipes/ivelt.com

Two successive runs have failed (I tried increasing available Memory and RAM between those two, but to no avail). Any idea what I could be missing there?

**Comment by rgaudin** [comment]
I don't think this website satisfies our criteria for our library. If one is not satisfied with the youzim.it limits, one can run zimit locally via Docker.

Btw the `Name` metadata is not correct.

As for the error, the crawler crashed. Indeed it mentions RAM

**Comment by rgaudin** [comment]
See https://github.com/webrecorder/browsertrix-crawler/issues/298

**Comment by benoit74** [comment]
Most probably already solved since months (not saying that the recipe succeeds, but memory issues are gone).
